docs/how-to/vector-stores/hybrid-search.md

```diff
@@ -0,0 +1,169 @@
+# Hybrid Search with Vector Stores
```

Comment:
You were supposed to replace the example in the existing "Hybrid Search with Vector Stores" document instead a creating a new alternative document with the same name.

Plus, for the document to actually be added to out documentation, you would need to include it in the mkdocs config.